apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/WriteTests.cs

```diff
@@ -52,148 +52,39 @@ public WriteTests(TestDatabaseFixture testFixture, ITestOutputHelper outputHelpe
         {
             using (var connection = await _testFixture.GetTestDatabaseConnectionAsync())
             {
-                values.Add("K", _lastKey = UniqueString(), SpannerDbType.String);
+                values.Add("K", SpannerDbType.String, _lastKey = UniqueString());
                 var cmd = connection.CreateInsertCommand("T", values);
 
                 return await cmd.ExecuteNonQueryAsync();
             }
         }
 
-        private async Task InsertAndRecordBytesAsync(byte[] bytes,
+        private async Task InsertAndRecordBytesAsync(
+            byte[] bytes,
             IDictionary<string, byte[]> record)
         {
             Assert.Equal(
                 1, await InsertAsync(
                     new SpannerParameterCollection
                     {
-                        new SpannerParameter("BytesValue", SpannerDbType.Bytes, bytes)
+                        new SpannerParameter("BytesValue", SpannerDbType.Bytes) {Value = bytes}
```

Comment:
nothing.  I missed undoing this as part of the undo left Value being set outside of the ctor.
Both ways are fine and the change can be undone.